PMM-11488 Correction. (#1029)
percona/pmm-doc: docs/get-started/query-analytics.md

## Query Analytics for PostgreSQL

> In QAN we do support two types of query sources, pg_stat_monitor and pg_stat_statements. In the past PGSS was the default query source, but since this version, it's now PGSM.

Reviewer: ```suggestion
In QAN we support two types of query sources, `pg_stat_monitor` and `pg_stat_statements`. Before this version, PGSS was the default query source, but now it's PGSM.
```
